.github/workflows/ci.yml

```
@@ -9,71 +9,37 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: 1.56.1
-          override: true
+      - uses: actions/checkout@v3
       - name: Run tests
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --verbose
+        run: cargo test --all-features --verbose --release --all
```

- We need `--release` in `zcash/librustzcash` because tests involving ZKPs are too slow in release mode. Here we don't have that restriction, so we should run tests in debug mode for both improved stack traces in the event of a crash, and faster CI.
- `--all` is a deprecated alias for `--workspace` (which we _do_ need here now that this repo is a workspace).
```suggestion
        run: cargo test --all-features --verbose --workspace
```

---

Using `--release` here also makes tests *much* faster; I routinely do this locally because it cuts the time of the proptests from minutes down to seconds.